src/rpc/blockchain.cpp

```diff
@@ -224,6 +224,18 @@ UniValue stakeToJSON(const CStakesDbEntry& stake)
     return result;
 }
 
+UniValue freeTxInfoToJSON(const CFreeTxInfo& freeTxInfo)
+{
+    AssertLockNotHeld(cs_main); // For performance reasons
+
+    UniValue result(UniValue::VOBJ);
+    result.pushKV("limit", static_cast<int>(freeTxInfo.getLimit()));
+    result.pushKV("used_blockchain_limit", static_cast<int>(freeTxInfo.getUsedConfirmedLimit()));
```

Comment:
<3

Reply:
pushKV accepts only signed integers, while we store it as unsigned. Either we add new functionality to legacy code, or do it this way.